src/graphics/program-lib/chunks/cookie.frag

```diff
@@ -37,5 +37,5 @@ vec3 getCookie2DClustered(sampler2D tex, mat4 transform, float intensity, bool i
     vec4 projPos = transform * vec4(vPositionW, 1.0);
     projPos.xy /= projPos.w;
     vec4 pixel = mix(vec4(1.0), texture2D(tex, projPos.xy), intensity);
-    return isRgb ? pixel.rgb : vec3(dot(pixel, cookieChannel));
+    return isRgb == true ? pixel.rgb : vec3(dot(pixel, cookieChannel));
```

Comment:
Why do you need to add the `== true` here?

Reply:
to be honest, I'm not sure, but have noticed one branch was not taken correctly unless I added it. So I added it to all. I could not find a different workaround.

Reply:
TBH, if I was refactoring this code, I would delete this. So if it's important, I would add a comment to say why it's there.